packages/core/admin/ee/server/services/seat-enforcement.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+const { take, drop } = require('lodash/fp');
+const { getService } = require('../../../server/utils');
+
+const enableUsersToLicenseLimit = async (numberOfUsersToEnable) => {
+  const data = await getService('user').getDisabledUserList();
+
+  if (!data?.value || data.value.length === 0) {
+    return;
+  }
+
+  const disabledUsers = JSON.parse(data.value);
```

Comment:
Same as before, I think it could be nice for `getDisabledUserList` to actually return the list of users.

So you could actually skip the parsing here:
```suggestion
  const disabledUsers = await getService('user').getDisabledUserList();

  if (!disabledUsers) {
    return;
  }

```